src/main/webapp/WEB-INF/views/cars/all.jsp

```diff
@@ -0,0 +1,33 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Display all cars</title>
+</head>
+<body>
+<h1>List of cars:</h1>
+<table>
+    <tr>
+        <td>ID</td>
+        <td>MODEL</td>
+        <td>MANUFACTURER NAME</td>
+        <td>MANUFACTURER COUNTRY</td>
+        <td>DRIVER NAME</td>>
+        <td>DRIVER LICENSE NUMBER</td>>
+    </tr>
+    <c:forEach items="${cars}" var="car">
+        <c:forEach items="${car.drivers}" var="driver">
```

Comment:
**OPTIONAL**: as I see, you duplicate cars' info when a car has multiple drivers. this should not happen on cars page, car is the main character on this page.

Reply:
I just don't know how it have to look like

Reply:
I would do it like this: we have a row in a table which represents the car, at the end of the row there is a list of drivers. In this way info about cars wont repeat